src/utils/asset.js

```diff
@@ -228,3 +230,16 @@ export const estimateGas = async ({ data, to, value }) => {
 
   return await provider.estimateGas(paramsForGasEstimate)
 }
+
+export const fetchTerraToken = async (address, network) => {
+  const { data: { [network]: tokens } } = await axios.get('https://assets.terra.money/cw20/tokens.json')
+  const token = tokens[address]
+  const { symbol, token: tokenAddress } = token
+  store.commit('TERRA_TOKEN', { symbol, tokenAddress })
```

Comment:
I don't think we need this state anymore right? Everything is stored fine on `customTokens`

Reply:
And can't this go in `tokenDetailProviders` like the rest of them?

Reply:
I see there are some slightly different requirements here compared to the others but we should just make the interface compatible

I think it will work if all the `getDetails` functions take both address and network